common.go

```diff
@@ -0,0 +1,26 @@
+package hazelcast
+
+import "context"
+
+func flagsSetOrClear(flags *int32, flag int32, enable bool) {
+	if enable {
+		*flags |= flag
+	} else {
+		*flags &^= flag
+	}
+}
+
+// extractLockID extracts lock ID from the context.
+// If the lock ID is not found, it returns the default lock ID.
+func extractLockID(ctx context.Context) int64 {
+	if ctx == nil {
+		return defaultLockID
+	}
+	if lockIDValue := ctx.Value(lockIDKey); lockIDValue == nil {
+		return defaultLockID
+	} else if lid, ok := lockIDValue.(lockID); !ok {
+		return defaultLockID
+	} else {
+		return int64(lid)
+	}
```

Comment:
How about rewriting this block in the following style?
```
lidv := ctx.Value(lockIDKey)
if lockIDValue == nil {
    return defaultLockID
}
lid, ok := lidv.(lockID)
if !ok {
   return defaultLockID
}
return int64(lid)
```

Reply:
Fixed as suggested